EDM-296 Publish helm charts under quay.io/flightctl/charts

mangelajo opened the pull request:
This is the standard with OCI helm charts. Consuming the charts from ArgoCD requires this, i.e. quay.io/flightctl cannot be supplied as base for charts.

celdrake commented on src/i18n.ts:
```diff
@@ -0,0 +1,27 @@
+  // for all options read: https://www.i18next.com/overview/configuration-options
+  .init({
+    fallbackLng: 'en',
+    debug: true,
```
Do you want to enable the debug for a one-off check?

rawagner replied:
Yep, this should be false. PR updated

celdrake reviewed (approved):
Tested with a browser set in Spanish and all works as expected.

mangelajo committed:
EDM-296 Publish helm charts under quay.io/flightctl/charts

This is the standard with OCI helm charts. Consuming the charts from
ArgoCD requires this, i.e. quay.io/flightctl cannot be supplied
as base for charts.

mangelajo commented on hack/publish_charts.sh:
```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+set -e
```
extra tool for manual publishing.

mangelajo commented:
updated the .sh permissions (+x exec)
